M2/Macaulay2/e/BasicPolyListParser.cpp

```diff
@@ -184,7 +217,7 @@ void parseBasicPoly(const std::string_view& str, const IdentifierHash& idenHash,
                 {
                   throw parsing_error("expected a digit at position " + std::to_string(begin_loc));
                 }
-              e = readInteger(str, begin_loc, end_loc);
+              e = readInteger_long(str, begin_loc, end_loc);
```

Comment:
exponent vectors are ints.  Are we silently truncating numbers to int size here?